cmd/podman/machine/inspect.go

```diff
@@ -92,6 +92,7 @@ func inspect(cmd *cobra.Command, args []string) error {
 			UserModeNetworking: false,
 			// TODO I think this should be the HostUser
 			Rootful: mc.HostUser.Rootful,
+			Rosetta: mc.Rosetta,
```

Comment:
we just scrutinized this struct as a team ... i dont believe this should be here. it is always false.  at the very least, it should be `omitempty` so it does not show for other platforms.

Reply:
I'm fine with leaving it there as long as it is `omitempty`

Reply:
fixed